### ISSUE TITLE
Duplicate rule names in `at-rule-no-unknown` message

### ISSUE THREAD

**Issue by ybiquitous** [kristerkari/stylelint-scss#480]
Hi there,

I found a bug that there are duplicate rule names in the message of the `at-rule-no-unknown` rule.

## Reproduction

Run the following commands on your shell:

```shell
# Install the packages
echo {} > package.json
npm i stylelint stylelint-scss
npm ls --depth 0
#=> /tmp/reproduction
#=> ├── stylelint@13.6.1
#=> └── stylelint-scss@3.18.0

# Prepare the config file
cat <<EOF >.stylelintrc.yml
plugins:
  - stylelint-scss

rules:
  at-rule-no-unknown: null
  scss/at-rule-no-unknown: true
EOF

# Preare the test file
echo '@unknown {}' > test.scss

# Run
npx stylelint test.scss
#=>
#=> test.scss
#=>  1:1  ✖  Unexpected unknown at-rule "@unknown" (at-rule-no-unknown)   scss/at-rule-no-unknown

# Run with JSON format
npx stylelint test.scss -f json
#=> [{"source":"/Users/masafumi.koba/tmp/foo2/test.scss","deprecations":[],"invalidOptionWarnings":[],"parseErrors":[],"errored":true,"warnings":[{"line":1,"column":1,"rule":"scss/at-rule-no-unknown","severity":"error","text":"Unexpected unknown at-rule \"@unknown\" (at-rule-no-unknown) (scss/at-rule-no-unknown)"}]}]
```

Here are the incorrect messages:

```
# In text format
Unexpected unknown at-rule "@unknown" (at-rule-no-unknown)

# In JSON format
Unexpected unknown at-rule "@unknown" (at-rule-no-unknown) (scss/at-rule-no-unknown)
```

I think we expect no ` (at-rule-no-unknown)`.

## Environment

```console
$ npm version
{
  npm: '6.14.5',
  ares: '1.16.0',
  brotli: '1.0.7',
  cldr: '37.0',
  icu: '67.1',
  llhttp: '2.0.4',
  modules: '83',
  napi: '6',
  nghttp2: '1.40.0',
  node: '14.3.0',
  openssl: '1.1.1g',
  tz: '2019c',
  unicode: '13.0',
  uv: '1.37.0',
  v8: '8.1.307.31-node.33',
  zlib: '1.2.11'
}
```

## Related code

https://github.com/kristerkari/stylelint-scss/blob/f54d9861e35891312bda98afe2404a993a4262e0/src/rules/at-rule-no-unknown/index.js#L32-L34

https://github.com/stylelint/stylelint/blob/a8042e7192cad993081cf982901948dbd5004048/lib/utils/ruleMessages.js#L23-L29

https://github.com/stylelint/stylelint/blob/a8042e7192cad993081cf982901948dbd5004048/lib/rules/at-rule-no-unknown/index.js#L16-L18

**Comment by ybiquitous** [comment]
This is an easy fix, so I've opened a PR #481.

**Comment by ybiquitous** [comment]
@kristerkari Thank you! 😊
